### ISSUE TITLE
Add blog posts of ScholarX mentees in 2019

### ISSUE THREAD

**Issue by janethavi** [sef-global/sef-site#949]
**Is your feature request related to a problem? Please describe.**
The ScholarX mentees in 2019 have created blog posts during the time of mentoring. But these blog posts are not mentioned on the website.

**Describe the solution you'd like**
Currently in the [ScholarX-19](https://sefglobal.org/scholarx/archive/2019/) page, there is a table for mentees. When a mentee is clicked, his/her LinkedIn page will be redirected. With the new suggestion, when a mentee is clicked, a card should be poped. This card should contain the mentee details, the blog post links along with the Linkedin URL. A design of the card is shown below. 
![blog-post (1)](https://user-images.githubusercontent.com/35697678/119947486-69350400-bfb5-11eb-91ba-bfbebae718f1.jpg)

The data(blog post URLs) can be found in [this document](https://docs.google.com/document/d/1nBI9pCteXv308W45pjg27FnaJAUxrOliC4_wwmMVzwo/edit?usp=sharing)

**Additional context**
To ease the implementation, You can refer to the [ScholarX page](https://sefglobal.org/scholarx/2021/) mentors cards. The logic and the implementation code snippets can be used from this page.

**Comment by Gravewalker666** [comment]
we aren't using blue coloured backgrounds right??

**Comment by janethavi** [comment]
> we aren't using blue coloured backgrounds right??

Sorry, I actually designed the card to show the structure. Anyways I updated the image above.

**Comment by YaiyaSam** [comment]
can I work on this?

**Comment by janethavi** [comment]
Sure @YaiyaSam go ahead!

**Comment by akshika47** [comment]
@janethavi I feel this design is a bit too clunky. Don't you think?

**Comment by janethavi** [comment]
Hi , 
I put this design because we currently have the exact same card in the [mentors section of ScholarX-21](https://sefglobal.org/scholarx/2021/). Will you be able to give a suggestion for a design?

**Comment by jayasanka-sack** [comment]
Maybe @YaiyaSam is already working on it, if not, let me put my two cents in. haha

We should definetly include these blogs on our page; it will works as a social proof. However I'm not sure whether we need to maintain something like a mentee profile.   

One suggestion would be - we could leverage on the feature stories section. So that we could load the blogs dynamically with their name. [ See attached image ] (Even we could rename the section's title; with something gives the idea that "this is what mentees think")

Another quick solution (quick, but might not ideal) would be to display them in another column. ex: [Blog 1](#), [Blog 2](#), [Blog 3](#)

Bottom line, 
the idea is to display the data in a single view without forcing user to click multiple places to access data.

![IMG_10CC1370BCA3-1](https://user-images.githubusercontent.com/33048395/120112975-f3d45980-c195-11eb-8c42-e3be3a1bbd0b.jpeg)

**Comment by YaiyaSam** [comment]
> Maybe @YaiyaSam is already working on it, if not, let me put my two cents in. haha
> 
> We should definetly include these blogs on our page; it will works as a social proof. However I'm not sure whether we need to maintain something like a mentee profile.
> 
> One suggestion would be - we could leverage on the feature stories section. So that we could load the blogs dynamically with their name. [ See attached image ] (Even we could rename the section's title; with something gives the idea that "this is what mentees think")
> 
> Another quick solution (quick, but might not ideal) would be to display them in another column. ex: [Blog 1](#), [Blog 2](#), [Blog 3](#)
> 
> Bottom line,
> the idea is to display the data in a single view without forcing user to click multiple places to access data.
> 
> ![IMG_10CC1370BCA3-1](https://user-images.githubusercontent.com/33048395/120112975-f3d45980-c195-11eb-8c42-e3be3a1bbd0b.jpeg)

 I didn't start yet 😄 Let me know what needs to be done!

**Comment by akshika47** [comment]
@jayasanka-sack I like the separate column idea, can someone create some mock UI first? When we have a separate section, we can have feature blog posts as well, but it is not a necessity.

**Comment by akshika47** [comment]
I just read the content of the wire-frame @jayasanka-sack  Lol

**Comment by janethavi** [comment]
Hello,

By taking @jayasanka-sack's suggestion and @akshika47's opinion, I redesigned the UI as suggested. So this will be the same table that we can find on the [ScholarX-19 page](https://sefglobal.org/scholarx/archive/2019/). 

**Explanation:-** I added a new column to display the blog names that each mentee has created. The blog name will be the link to the original article.

**Please Note:-** When there are several blogs created by an individual mentee the row height of the table will be increased.

![Untitled Diagram](https://user-images.githubusercontent.com/35697678/120496136-f247a300-c3da-11eb-8eec-fca77e1f0364.jpg)
